rabbitmq_status.py

```diff
@@ -93,8 +93,15 @@ def main():
 
     if r.ok:
         resp_json = r.json()  # Parse the JSON once
-        if any('channels' in connection and connection['channels'] > 1 for connection in resp_json):
-            status_err('Detected RabbitMQ connections with multiple channels. Please check RabbitMQ and all Openstack consumers')
+        # We routinely see connection['channels'] == 2, so we cannot alarm if
+        # connection['channels'] > 1.  Note that connection['channels'] does
+        # not appear to be the total number of channels using the connection,
+        # but rather the channel number which identifies it within a
+        # connection.  We'll assume if we have a channel w/ number 11 or
```

Comment:
A connection can have multiple channels, and each channel has a "number", which is its index in the channel list for that connection, starting at 1. When querying the management API from the channel perspective (channel["number"]), we see the index of the channel being queried; from the connection perspective (connection["channels"]), we see the count of channels for that connection.

As @mattt416 and I discovered today, it's fairly common for there to be a couple of channels (at least briefly) for openstack component connections. The original concern Support had was with broken rabbit/clients causing ever-increasing channel creation, and the monitor was intending to catch that by making sure there was never a channel with an index ("number") > 1. So, instead, this change is attempting to put an arbitrary but likely reasonable limit on channel ballooning, without assuming that component connections will only ever use a single channel (which they demonstrably do not).